packages/worker/src/workflow-log-interceptor.ts

```diff
@@ -74,4 +74,5 @@ export class WorkflowInboundLogInterceptor implements WorkflowInboundCallsInterc
   }
 }
 
+// ts-prune-ignore-next
 export const interceptors: WorkflowInterceptorsFactory = () => ({ inbound: [new WorkflowInboundLogInterceptor()] });
```

Comment:
Or just add this to the list of entrypoints in the prune tsconfig.. fine either way I guess..